runtime/onert/core/src/exec/Execution.test.cc

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-#include <gtest/gtest.h>
-#include <thread>
+#include "exec/Execution.h"
```

Comment:
(optional) Following may be more consistent to others.
```suggestion
#include "Execution.h"
```